.gitignore

```diff
@@ -1,3 +1,11 @@
 *~
 .dub/
 target/
+*.o
+*.s
+*.a
+.DS_Store
+json/test_asdf
```

Comment:
All intermediate/output binaries should be located in "target", please remove those extra rules and update Makefile accordingly. The only additional .gitignore rule I see would be ".DS_Store".

Reply:
Will do